src/res_core.ml

```diff
@@ -1534,12 +1534,6 @@ and parseParameter p =
   then
     let startPos = p.Parser.startPos in
     let uncurried = Parser.optional p Token.Dot in
-    (* two scenarios:
```

Comment:
Can we move this comment/mental model to the appropriate place?

Reply:
https://github.com/rescript-lang/syntax/pull/683/commits/f2a972059e3a25002a6181daaf682724f7cee8c6 How about this?